### PR TITLE
SDPA-346

### PR DESCRIPTION
**Pull request by darconny**
- UI improvements for image edit/crop modal
- css refactoring

### CODE REVIEW

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/superdesk/upload/styles/upload.less [hunk 1]
```diff
@@ -580,87 +580,93 @@ media uploader
...
-            font-size: 11px;
-            text-transform: uppercase;
-            color: @grayLight;
-        }
-    }
+.modal-body__flex {
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+	overflow-y: hidden !important;
```
Comment (vied12) [inline scripts/superdesk/upload/styles/upload.less]: please avoid `!important`

File: styles/less/layouts.less [hunk 2]
```diff
@@ -6,29 +6,27 @@
...
 body {
-    margin: 0;
-    font-family: @baseFontFamily;
-    font-size: @baseFontSize;
-    line-height: @baseLineHeight;
-    color: @sd-text;
-    background-color: @sd-background;
-    overflow: hidden;
-    height:100%;
+	margin: 0;
```
Comment (vied12) [inline styles/less/layouts.less]: we indent with spaces :)
please configure your editor

File: scripts/superdesk/upload/styles/upload.less (hunk 1 above)
Comment (darconny) [inline scripts/superdesk/upload/styles/upload.less]: this is here to overwrite overflow:auto; that comes from the standard body, and we don't need it here for flex implementation

Comment (vied12) [inline scripts/superdesk/upload/styles/upload.less]: then as I told you in another PR, it's better to play with selector precision than `important!`

Comment (darconny) [inline scripts/superdesk/upload/styles/upload.less]: In this case I do not agree.

Comment (vied12) [inline scripts/superdesk/upload/styles/upload.less]: https://css-tricks.com/when-using-important-is-the-right-choice/ ;)

**Pull request merged**